packages/synchro-charts/src/components/sc-dial/sc-dial-base/svg-dial-svg.spec.tsx

```diff
@@ -153,17 +199,22 @@ describe('size', () => {
     const unit = dialSvg.querySelectorAll('tspan')[1].attributes.getNamedItem('font-size')?.value;
     expect(unit).toEqual(`${UNIT_SIZE}`);
 
-    const circle = dialSvg.querySelector('circle')?.attributes.getNamedItem('stroke-width')?.value;
-    expect(circle).toEqual(`${DIAL_THICKNESS}`);
-
-    const iconWidth = dialSvg.querySelector('svg')?.attributes.getNamedItem('width')?.value;
-    const iconHeight = dialSvg.querySelector('svg')?.attributes.getNamedItem('height')?.value;
+    const iconWidth = dialSvg.querySelectorAll('svg')[1]?.attributes.getNamedItem('width')?.value;
+    const iconHeight = dialSvg.querySelectorAll('svg')[1]?.attributes.getNamedItem('height')?.value;
     expect(iconWidth).toEqual(`${ICON_SIZE}px`);
     expect(iconHeight).toEqual(`${ICON_SIZE}px`);
+
+    setTimeout(() => {
+      const path = dialSvg.querySelectorAll('path');
+      const pathDefault = path[0]?.attributes.getNamedItem('d')?.value;
+      const pathColor = path[1]?.attributes.getNamedItem('d')?.value;
+      expect(pathDefault).toContain(DIAMETER - DIAL_THICKNESS);
+      expect(pathColor).toContain(DIAMETER - DIAL_THICKNESS);
+    }, 1000);
```

Comment:
these test assertions won't actually execute. avoid arbitrary timeouts

Reply:
Has Changed to get path through `arc` function, and then judged if expect `path` contain thickness.